plane/source/docs/reverse-thrust-setup.rst

```diff
@@ -0,0 +1,71 @@
+.. _reverse-thrust-setup:
+
+
+====================
+Reverse Thrust Setup
+====================
+
+Reverse thrust can be used in Plane for any auto-throttle mode and in automatic landings, and/or under pilot control to provide steeper descents without gaining airspeed.
+
+In order to use reverse thrust, the ESC must be capable of it. Usually, the ESC input to output will be symmetric with equal forward and reverse output versus input around its input center point.
```

Comment:
```suggestion
In order to use reverse thrust, the ESC must be capable of it. If so, usually the ESC input to output will be symmetric with equal forward and reverse output. For example, the center point 1500 would be idle/stopped instead of the half forward throttle.
```